2_Growth-Maturity-Decline.md

```diff
@@ -1,15 +1,33 @@
 # Metric: Growth-Maturity-Decline
 
 
+## Issue Resolution
  Activity Metric | Description
  --- | ---
- Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time
- Bug Time to Close | Time from bug opened to bug closed over time
- Time to Merge | Time from submission to merge or abandon over time
- Review Efficiency | Number of merged patches / Number of abandoned patches over time
- Number of New Contributors | Number of first time contributors over time
- Community Growth | Number of new developers over time
- Community Toxicity | Toxic behavior and language over time
+ Open issues | Total number of open issues.
+ Closed issues | Total number of closed issues.
+ Resolution efficiency | Number of closed issues / number of abandoned issues.
+ Average & median time of open issues | The average & median amount of time issues remain open.
+ Average & median time of first response | The average & median amount of time until the first response to an issue is made.
+ Distribution of open issues over time | Proportional frequency of open issue time duration.
+ Distribution of issue resolution over time | Proportional frequency of closed issue time duration.
+
+## Code Development
+ Activity Metric | Description
+ --- | ---
+ Number of commits | Total number of commits.
+ Number of lines changed | Total number of lines of code that have been changed.
+ Average & median time to merge code | The average & median amount of time it takes to get code merged upstream. 
+ Distribution of time to merge | Proportional frequency of code merge to upstream time duration.
+ Review efficiency | Number of merged patches / number of abandoned patches.

```

Comment:
Is this a metric across all of time or is this ratio for a specific time window (e.g. submitted over 30 days)?